Yevgeny updated people

yevgenyr opened the pull request:
Updated my personal info in People db

sbillinge commented:
Thanks so much for this @yevgenyr.  I will have to modify regolith to accept the linked_in ID in the schema, but I can do that quickly then it should pass CI....  It is a good addition.

sbillinge commented on db/people.yml:
```diff
@@ -4075,14 +4075,37 @@ yrakita:
+    - Yevgeny Rakita Shlafstein
   avatar: https://github.com/yevgenyr.png
-  bio: bio
+  bio: Yevgeny Rakita (Shlafstein) grew up in Israel, and after finishe his PhD, 
```
typo

sbillinge reviewed (commented):
one typo.  May as well catch it.... :)

sbillinge commented:
Close and reopen to trigger CI after modifying regolith

sbillinge commented:
It looks as if there is still an issue with the entry:
```
could not find expected ':'

  in "../db/people.yml", line 4083, column 3
```

yevgenyr commented:
please try now. added double quotes and fixed typo. I left linkedin. Let me know if it is better to remove it for now and add when things are ready to host it

sbillinge commented:
Thanks @yevgenyr I will merge like this.

I see from the deleted part that  the problem was just an indentation problem.  The text from line 2 onwards should have been indented a further two whitespaces.  No quotes etc. were actually needed, but I think it is fine as is (the html is building but I haven't looked how it renders).
